src/Components/SmartComponents/SystemCves/SystemCves.js

```diff
@@ -25,18 +25,32 @@ import { useCreateUrlParams } from '../../../Helpers/MiscHelper';
 export const CVETableContext = createContext({});
 
 export const SystemCVEs = (props) => {
+
     const dispatch = useDispatch();
     const [StatusModal, setStatusModal] = useState(() => () => null);
-    const systemCVEs = useSelector(({ SystemCvesStore }) => SystemCvesStore.cveList);
-    const parameters = useSelector(({ SystemCvesStore }) => SystemCvesStore.parameters);
-    const selectedCves = useSelector(({ SystemCvesStore }) => SystemCvesStore.selectedCves);
-    const openedCves = useSelector(({ SystemCvesStore }) => SystemCvesStore.openedCves);
-    const expandCveDescription = useSelector(({ SystemCvesStore }) => SystemCvesStore.expandCveDescription);
+
+    const systemCVEs = useSelector(
+        ({ SystemCvesStore }) => SystemCvesStore.cveList
+    );
+    const parameters = useSelector(
+        ({ SystemCvesStore }) => SystemCvesStore.parameters
+    );
+    const selectedCves = useSelector(
+        ({ SystemCvesStore }) => SystemCvesStore.selectedCves
+    );
+    const openedCves = useSelector(
+        ({ SystemCvesStore }) => SystemCvesStore.openedCves
+    );
+    const expandCveDescription = useSelector(
+        ({ SystemCvesStore }) => SystemCvesStore.expandCveDescription
+    );
+
     const cves =  useMemo(() => createCveListBySystem(props.entity, systemCVEs), [systemCVEs]);
     const [createUrlParams, urlParameters] = useCreateUrlParams(CVES_ALLOWED_PARAMS);
 
     useEffect(() =>{
         !parameters.isFirstMount && dispatch(fetchCveListBySystem({ ...parameters, system: props.entity }));
+        parameters.urlParamsAllowed && createUrlParams({ ...parameters });
```

Comment:
what is the `urlParamsAllowed` for?

Reply:
I added urlParamsAllowed to solve [VULN-956](https://projects.engineering.redhat.com/browse/VULN-956). When the page loads for the first time we should not push new URL parameters to history. It will cause double click problem on navigation.
          At first urlParamsAllowed is false, which will prevent calling  createUrlParams. After first load this bool value is set to false to allow pushing new URL parameters to history. If you think that there is better way, I appreciate suggestions. 

This solved what you have mentioned in previous PR: 'you load a page bypassing the URL then you need to click back twice in order to go to the previous page.'

Reply:
> When the page loads for the first time

I understand that the flag `isFirstMount` determines the same?  isn't it? why do you need another flag then?

Reply:
I was also thinking one flag would enough. Then I realised, one flag will create another bug. Because there is inventory as well. As you know inventory fires onRefresh function call at the beginning. I need to somehow detect this first call is not coming from inventory.